packages/analytics-browser/src/browser-client.ts

```diff
@@ -37,7 +37,22 @@ export class AmplitudeBrowser extends AmplitudeCore implements BrowserClient {
   previousSessionDeviceId: string | undefined;
   previousSessionUserId: string | undefined;
 
-  init(apiKey = '', userId?: string, options?: BrowserOptions) {
+  init(apiKey = '', userIdOrOptions?: string | BrowserOptions, maybeOptions?: BrowserOptions) {
```

Comment:
Change looks good to me. Just to confirm, should we apply the similar change to [react-native](https://github.com/amplitude/Amplitude-TypeScript/blob/dcf21ca0a2603c042076b6a5d02ea3f184c1ed8c/packages/analytics-react-native/src/react-native-client.ts#L45-L47) one as well for consistency?

Reply:
No need to be in single PR for sure, thank you!